### DISCUSSION TITLE
Vcpkg Support.

### DISCUSSION THREAD

**Discussion by bowenxuuu** [slint-ui/slint#6085] [Ideas]
Hi, 

Recently, I developed an internally-used software application with `Rust + Slint`, and it runs well.
Moving forward, I plan to develop another application with `Cpp + Slint`, as `Cpp` has some mature third-party libraries that I need. So, I would like to ask if the Slint team has any plans to support `Vcpkg`? This would make integrating Slint with Cpp more convenient.
﻿
Thank you!

**Comment by ogoffart** [comment]
Yes, we would like to.
Upstream issue: https://github.com/microsoft/vcpkg/issues/33824
But it depends on the ability to package rust dependencies: https://github.com/microsoft/vcpkg/issues/20619
I'm not familiar enough with vcpkg to know if it could be done without this issue being fixed (eg, by using the binary packages, or doing the necessary things in the vcpkg receipt)
Any help to get it there would be welcome.
